src/main/java/com/coyoapp/tinytask/dto/TaskResponse.java

```diff
@@ -14,4 +14,6 @@
   private String id;
 
   private String name;
+
```

Comment:
due to the adjustments in the DefaultFileService it is more a unique file identifier instead of the origin filename